src/Integration.Vsix/SonarLintTagger/SonarLanguageRecognizer.cs

```diff
@@ -61,15 +61,17 @@ public IEnumerable<SonarLanguage> Detect(ITextDocument textDocument, ITextBuffer
 
             // Languages are for now mainly exclusive but it should possible for the same file to be analyzed by multiple
             // plugins (language plugin).
+            var detectedLanguages = new List<SonarLanguage>();
```

Comment:
Miscellaneous: unnecessary use of _yield_ (slower at runtime, and makes debugging harder)